Tests/TuistLoaderTests/Loaders/CachedManifestLoaderTests.swift

```diff
@@ -129,11 +140,29 @@ final class CachedManifestLoaderTests: TuistUnitTestCase {
         XCTAssertEqual(recordedLoadProjectCalls, 2)
     }
 
+    func test_load_pluginsHashChanged() throws {
```

Comment:
This test fails in the current main branch

Reply:
worth adding another test case for the scenario where plugins were unchanged thus only loading plugins once _(i.e. caching in effect)_